src/main/java/org/opensearch/agent/tools/SearchAnomalyDetectorsTool.java

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.agent.tools;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.opensearch.action.search.SearchRequest;
+import org.opensearch.action.search.SearchResponse;
+import org.opensearch.ad.client.AnomalyDetectionNodeClient;
```

Comment:
Test is failing saying it can't find this.

Reply:
Right. It is because of the difficulty in having passing builds in 3.0 at all, especially with all of the added plugin dependencies. See comment above https://github.com/opensearch-project/skills/pull/39#issuecomment-1864815319

Reply:
To prevent friction and keep things moving along, prefer to enforce on 2.x only. There is lots of changes happening in dependent repos that are inevitably going to keep breaking things here given the dependency model.

Reply:
We need to set up end to end testing and my understanding is that will be on `main`.  Merging this would prevent any other CI checks from completing on any future PRs here.